packages/types/lib/db.ts

```diff
@@ -7,5 +7,10 @@ export interface Deleted {
     deleted_at?: Date | null;
     deleted?: boolean;
 }
+export interface DeletedCorrect {
+    deleted_at: Date | null;
+    deleted: boolean;
+}
```

Comment:
annoyingly deleted and deleted_at were also set to undefined but only realised that now so all previous DB* types are kind of wrong too